src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,23 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <UserInfo user={user} />
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
user can be undefined, so better to add the check here